sklearn/utils/__init__.py

```diff
@@ -79,6 +81,31 @@
 _IS_32BIT = 8 * struct.calcsize("P") == 32
 
 
+def in_unstable_openblas_configuration():
```

Comment:
Maybe make this more specific and private: `_unstable_arm_openblas_version`.

Reply:
OK for making it private.

Potentially, there might be other unstable configurations (in the future) that we could detect and add here, I don't think that we need to be specific here.

Also, to me, it makes sense to be responsible for problem prevention even if they arise from dependencies, and I think plugging this check as a guard on for `FastSquaredEuclidean*.is_usable_for` would make the implementations safer and not necessarily more complex. WDYT?

Reply:
As you wish.